Screenbox/Controls/CommonGridViewItem.xaml

```diff
@@ -19,23 +19,24 @@
         <Grid
             Grid.Row="0"
             Height="{x:Bind ThumbnailHeight, Mode=OneWay}"
-            Background="{ThemeResource SolidBackgroundFillColorSecondaryBrush}"
             BorderBrush="{ThemeResource ControlElevationBorderBrush}"
             BorderThickness="1"
-            CornerRadius="{x:Bind CornerRadius}">
-            <IconSourceElement
-                x:Name="PlaceholderIcon"
-                HorizontalAlignment="Center"
-                VerticalAlignment="Center"
-                Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                IconSource="{x:Bind PlaceholderIconSource, Mode=OneWay}"
-                Visibility="{x:Bind ThumbnailSource, Converter={StaticResource InverseEmptyObjectToVisibilityConverter}, Mode=OneWay}" />
-            <Border CornerRadius="{x:Bind CornerRadius}">
-                <Image
+            CornerRadius="{x:Bind CornerRadius}"
+            FlowDirection="LeftToRight">
```

Comment:
Wouldn't this force LeftToRight to the child element as well? In this case, wouldn't the icon be flipped?

Reply:
> Wouldn't this force LeftToRight to the child element as well? In this case, wouldn't the icon be flipped?

Yes. No, there's no change in the icons direction as they are already "forced" to LeftToRight.